k8s/demo/common/ccd/definition-store-api.yaml

```diff
@@ -19,12 +19,24 @@ spec:
   values:
     java:
       replicas: 2
+      keyVaults:
+        ccd:
+          secrets:
+            - definition-store-api-POSTGRES-PASS-V11
+            - ccd-definition-s2s-secret
+            - ccd-ELASTIC-SEARCH-URL
+            - ccd-ELASTIC-SEARCH-PASSWORD
+            - storage-account-primary-connection-string
+            - AppInsightsInstrumentationKey
       useInterpodAntiAffinity: true
       image: hmctspublic.azurecr.io/ccd/definition-store-api:pr-575-51aab560
       environment:
         IDAM_USER_URL: https://idam-web-public.demo.platform.hmcts.net
         DEFINITION_STORE_DB_OPTIONS: "?sslmode=require&gssEncMode=disable"
         DEFINITION_STORE_DB_MAX_POOL_SIZE: 32
+        DEFINITION_STORE_DB_NAME: ccd_definition_store-v11
+        DEFINITION_STORE_DB_HOST: ccd-definition-store-api-postgres-db-v11-demo.postgres.database.azure.com
+        DEFINITION_STORE_DB_USERNAME: ccd@ccd-definition-store-api-postgres-db-v11-demo
         DEFINITION_STORE_S2S_AUTHORISED_SERVICES: ccd_data,ccd_gw,ccd_admin,jui_webapp,pui_webapp,aac_manage_case_assignment
     global:
       environment: demo
```

Comment:
Other microservices are going to be added once we prove connectivity to the new DB-v11 using v11 credentials